doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -984,10 +984,12 @@ Helper tactics
 
 .. tacn:: decide equality
 
-   Solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}`,
-   where :g:`R` is an inductive type such that its constructors do not take
-   proofs or functions as arguments, nor objects in dependent types. It
-   solves goals of the form :g:`{x = y} + {~ x = y}` as well.
+   Solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}` or
+   :g:`forall x y : R, (x = y) \/ (~ x = y)`, where :g:`R` is an
+   inductive type such that its constructors do not take proofs
+   or functions as arguments, nor objects in dependent types. It
+   solves goals of the form
+   :g:`{x = y} + {~ x = y}` or :g:`(x = y) \/ (~ x = y)` as well.
```

Comment:
Just curious: what will this tactic do with a goals in the form `{~ x = y} + {x = y}`?  If it doesn't handle it, maybe add a sentence on how transform it to the form it can handle if that's simple but not totally obvious.

Reply:
I just tried, and it does handle goals in this form.

Reply:
I almost missed your suggestion above…  I will apply it.